examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter

```diff
@@ -5264,6 +5264,8 @@ cluster ApplicationLauncher = 1292 {
     kSuccess = 0;
     kAppNotAvailable = 1;
     kSystemBusy = 2;
+    kStatusPending = 3;
+    kStatusInstalling = 4;
```

Comment:
this will be blocked on us getting a CCB approved by Matter WG. Once we know this is the best approach, I can kick off that process but this is an FYI.

Reply:
ack